src/target.zig

```diff
@@ -45,7 +45,8 @@ pub fn requiresPIC(target: std.Target, linking_libc: bool) bool {
     return target.isAndroid() or
         target.os.tag == .windows or target.os.tag == .uefi or
         osRequiresLibC(target) or
-        (linking_libc and target.isGnuLibC());
+        (linking_libc and target.isGnuLibC()) or
+        (target.abi == .ohos and target.cpu.arch == .aarch64);
```

Comment:
Openharmony support `arm64-v8a` `armeabi-v7a` `x86_64`, so should we remove ` target.cpu.arch == .aarch64` ?

Reply:
I don't know, this just matches what Clang does https://github.com/llvm/llvm-project/blob/6733a505a1afb8eb4db2f6d85426d79ff0dc5eee/clang/lib/Driver/ToolChains/CommonArgs.cpp#L1735-L1737

Reply:
ok thanks for your work, i will try to test it.